lib/tasks/publishing_api.rake

```diff
@@ -188,6 +188,18 @@ namespace :publishing_api do
     puts "Finished queuing items for Publishing API"
   end
 
+  desc "Republish all documents with official attachments"
+  task republish_documents_with_official_attachments: :environment do
+    attachment_ids = Attachment.where("unnumbered_command_paper = true OR command_paper_number != '' OR unnumbered_hoc_paper = true OR hoc_paper_number != ''").pluck(:attachable_id).uniq
```

Comment:
Minor - these aren't attachment_ids but are attachable_ids. If we want to be sure these are edition ids we could do:

```
edition_ids = Attachment.where("unnumbered_command_paper = true OR command_paper_number != '' OR unnumbered_hoc_paper = true OR hoc_paper_number != ''")
                        .where(attachable_type: "Edition")
                        .distinct
                        .pluck(:attachable_id)
```
Which would also save having to do the uniq after a query.